src/components/textToSpeech/js/TextToSpeech.js

```diff
@@ -72,9 +72,11 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
             onSpeechQueued: null
         },
         members: {
-            queue: []
+            queue: [],
+            //
+            currentUtterance: {}
```

Comment:
Yes, that's right - the queue has nothing to do with your throttling, but it's still an appropriate place for the utterance. Note that it is created one for one with each queue element: https://github.com/fluid-project/infusion/blob/master/src/components/textToSpeech/js/TextToSpeech.js#L188